docs/astrocut/index.rst

```diff
@@ -405,20 +408,20 @@ cutout location/size(s) necesary to cover the entire path.
 Combining cutouts
 -----------------
 
-The `~astrocut.CutoutsComibner` class allows the user to take one or more Astrocut cutout
+The `~astrocut.CutoutsCombiner` class allows the user to take one or more Astrocut cutout
 FITS files (as from  `~astrocut.fits_cut`) with a shared WCS object, and combine them into
 a single cutout. In practical terms this means that you should make the same cutout in the
 all of the images you want to combine.
```

Comment:
It's not clear to me what this last sentence means. @jaymedina - Any thoughts on meaning and/or how to make it more clear?

Reply:
I think this last sentence is trying to say you should request the same cutout size in all the images you want to combine